### PR TITLE
Jb49305

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last two commits belong to this PR

File: src/tools/sdktool/addsfdkbuildengineoperation.h [hunk 1]
```diff
@@ -59,6 +59,7 @@ class AddSfdkBuildEngineOperation : public Operation
            const QUrl &vmUri,
            const QDateTime &creationTime,
            bool autodetected,
+           const QString &sharedInstallPath,
```
Comment (vigejolla) [inline src/tools/sdktool/addsfdkbuildengineoperation.h]: Nothing wrong with this, but now RmSfdkBuildEngineOperation::test() won't compile anymore, as it doesn't have this parameter

**Comment by vigejolla** [comment]
Nice, just fix the tests :)

File: src/tools/sdktool/addsfdkbuildengineoperation.h (hunk 1 above)
Comment (martyone) [reply src/tools/sdktool/addsfdkbuildengineoperation.h]: oops, fixed.

**Pull request merged**